src/components/layout/nav/nav-careers.tsx

```diff
@@ -116,33 +133,21 @@ export const NavCareers = ({ is_besquare }) => {
                                     loading="eager"
                                 />
                             </CareerLogo>
-                            <CareerLink
-                                active={current_page === 'home'}
-                                activeClassName="active"
-                                to="/careers/"
-                                aria-label={'Careers'}
-                                partiallyActive={true}
-                            >
-                                HOME
-                            </CareerLink>
-                            <CareerLink
-                                active={current_page === 'locations'}
-                                activeClassName="active"
-                                to="/careers/locations/"
-                                aria-label={'Locations'}
-                                partiallyActive={true}
-                            >
-                                LOCATIONS
-                            </CareerLink>
-                            <CareerLink
-                                active={current_page === 'besquare'}
-                                activeClassName="active"
-                                to="/careers/besquare/"
-                                aria-label={'BeSquare'}
-                                partiallyActive={true}
-                            >
-                                BESQUARE
-                            </CareerLink>
+
+                            {links.map((item) => {
```

Comment:
don't need to use return 
you can simply remove `{ return (` and just write `(item) => <CareerLink ...`